build_process/package-management.md

```diff
@@ -80,14 +80,14 @@ $ npm publish
 
 If we plan to run versioning from CI, it may require creating a [machine user][machine] to setup ssh/gpg key for pushing and [signing versions][versionSign]. This can allow the CI system to make changes without attributing them to one team member.
 
-Consider publishing [pre-release][prerelease] versions when we need time to validate changes before promoting to a proper version. Pre-release versions can incorporate a semver compliant identifier. A good technique is to use the current git sha as an id to avoid repeated versions.
+We generally [pre-release][prerelease] versions when we need time to validate changes before promoting to a proper version. Pre-release versions can incorporate a semver compliant identifier. A good technique is to use the current git sha as an id to avoid repeated versions.
 
-**Please don't republish package versions**. Some private registries can be configured to allow republishing of package versions. Overwriting an existing package can lead to failed checksums when downloading or frustrating caching issues with the local npm cache.
+**We don't republish package versions**. Some private registries can be configured to allow republishing of package versions. Overwriting an existing package can lead to failed checksums when downloading or frustrating caching issues with the local npm cache.
 
 ## Security
 Registry accounts should enable [two-factor auth][2fa]. This can be enabled for user login [as well as publishing][2faPublish].
```

Comment:
We'd like the voice of the Standard to be descriptive rather than prescriptive, so "what we do" rather than "what we should do." Because of that, I would like to remove the word "should" here. But I'm not confident I could rewrite this and keep the original meaning. Could you rewrite this to remove the word "should"?